guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -167,7 +167,13 @@ Updating the metadata in `composer.json` file is entirely superficial, not funct
 composer update
 ```
 
-### Clear caches and generated content
+### Clean the Magento configuration cache
+
```

Comment:
Would it be worth adding a tidbit of info here regarding clearing the cache after upgrades?

Reply:
Done. I think that was a copy and paste error.